packages/shopware-6-client/src/services/contextService.ts

```diff
@@ -0,0 +1,75 @@
+import { Currency } from "../interfaces/models/system/currency/Currency";
+import { apiService } from "../apiService";
+import { config } from "../settings";
+import {
+  getContextCurrencyEndpoint,
+  getContextCountryEndpoint,
+  getContextPaymentMethodEndpoint,
+  getContextShippingMethodEndpoint,
+  getContextLanguageEndpoint
+} from "../endpoints";
+import { Country } from "../interfaces/models/system/country/Country";
+import { ShippingMethod } from "../interfaces/models/checkout/shipping/ShippingMethod";
+import { PaymentMethod } from "../interfaces/models/checkout/payment/PaymentMethod";
+import { Language } from "../interfaces/models/framework/language/Language";
+import { SearchResult } from "../interfaces/response/SearchResult";
+
+const getCurrencies = async function(): Promise<SearchResult<Currency[]>> {
+  const resp = await apiService.get(
+    config.endpoint + getContextCurrencyEndpoint()
+  );
+
+  return resp.data;
+};
+
+const getLanguages = async function(): Promise<SearchResult<Language[]>> {
+  const resp = await apiService.get(
+    config.endpoint + getContextLanguageEndpoint()
+  );
+
+  return resp.data;
+};
+
+const getCountries = async function(): Promise<SearchResult<Country[]>> {
+  const resp = await apiService.get(
+    config.endpoint + getContextCountryEndpoint()
+  );
+
+  return resp.data;
+};
+
+const getPaymentMethods = async function(): Promise<
+  SearchResult<PaymentMethod[]>
+> {
+  const resp = await apiService.get(
+    config.endpoint + getContextPaymentMethodEndpoint()
+  );
+
+  return resp.data;
+};
+
+const getShippingMethods = async function(): Promise<
+  SearchResult<ShippingMethod[]>
+> {
+  const resp = await apiService.get(
+    config.endpoint + getContextShippingMethodEndpoint()
+  );
+
+  return resp.data;
+};
+
+interface ContextService {
+  getCurrencies: () => Promise<SearchResult<Currency[]>>;
+  getLanguages: () => Promise<SearchResult<Language[]>>;
+  getCountries: () => Promise<SearchResult<Country[]>>;
+  getPaymentMethods: () => Promise<SearchResult<PaymentMethod[]>>;
+  getShippingMethods: () => Promise<SearchResult<ShippingMethod[]>>;
```

Comment:
```suggestion
  getAvailableCurrencies: () => Promise<SearchResult<Currency[]>>;
  getAvailableLanguages: () => Promise<SearchResult<Language[]>>;
  getAvailableCountries: () => Promise<SearchResult<Country[]>>;
  getPaymentMethods: () => Promise<SearchResult<PaymentMethod[]>>;
  getAvailableShippingMethods: () => Promise<SearchResult<ShippingMethod[]>>;
```
having those methods in cotext is totally fine for me, it's directly related, but under the word `context` we should rather have something related to the customer's current session. 
We could have all those methods with the keyword "available" and also with the "current" word when the SW Core Team makes the methods that responses the current context's details so we don't have any at the moment.